client/helpers.go

```diff
@@ -238,22 +238,15 @@ func nearExpiry(r *data.SignedRoot) bool {
 }
 
 // Fetches a public key from a remote store, given a gun and role
-func getRemoteKey(url, gun, role string, rt http.RoundTripper) (data.PublicKey, error) {
+func getRemoteKey(url, gun, role string, rt http.RoundTripper, rotate bool) (data.PublicKey, error) {
```

Comment:
It seems that it might make it clearer to rename and update the description for this method, since it can also rotate the remote key if specified?